client/src/partials/inventory/list/list.js

```diff
@@ -0,0 +1,138 @@
+angular.module('bhima.controllers')
+.controller('InventoryListController', InventoryListController);
+
+// dependencies injection
+InventoryListController.$inject = [
+  '$translate', 'InventoryService', 'NotifyService', 'uiGridConstants', 'ModalService'
+];
+
+/**
+ * Inventory List Controllers
+ * This controller is responsible of the inventory list module
+ */
+function InventoryListController ($translate, Inventory, Notify, uiGridConstants, Modal) {
+  var vm = this;
+
+  /** gobal variables */
+  vm.filterEnabled = false;
+  vm.gridOptions = {};
+  vm.gridApi = {};
+
+  /** paths in the headercrumb */
+  vm.bcPaths = [
+    { label : 'TREE.INVENTORY' },
+    { label : 'TREE.INVENTORY_LIST' }
+  ];
+
+  /** buttons in the headercrumb */
+  vm.bcButtons = [
+    { icon: 'fa fa-filter', label: $translate.instant('FORM.BUTTONS.FILTER'), action: toggleFilter, color: 'btn-default' },
+    { icon: 'fa fa-plus', label: $translate.instant('FORM.LABELS.ADD'), action: addInventoryItem, color: 'btn-primary' },
+    { icon: 'fa fa-print', label: $translate.instant('FORM.LABELS.PRINT'), action: printList, color: 'btn-default' }
+  ];
+
+  // edit button template
+  var editTemplate = '<div class="text-center">' +
+    '<button class="btn btn-sm btn-default" ng-click="grid.appScope.editInventoryItem(row.entity)">' +
+    '<i class="fa fa-edit"></i> ' +
+    '{{ "FORM.LABELS.EDIT" | translate }}' +
+    '</button></div>';
+
+  // consumable icon template
+  var iconTemplate = '<div class="text-center">' +
```

Comment:
These buttons are really big...  Could you make them look like the `patient invoice` module's indicators?  Those are a much more reasonable size, I think.